components/script/dom/element.rs

```diff
@@ -746,6 +746,30 @@ impl LayoutElementHelpers for LayoutJS<Element> {
                 shared_lock,
                 PropertyDeclaration::BorderRightWidth(width_value)));
         }
+
+        let cellpadding = if let Some(this) = self.downcast::<HTMLTableCellElement>() {
```

Comment:
We should seriously make this function virtual, it's really branchy already...

Of course not your fault.